cmake/PythonSupport.cmake

```diff
@@ -11,7 +11,7 @@
 #    SPDX-License-Identifier: BSD-2-Clause.degroot
 #    License-Filename: LICENSES/BSD-2-Clause.degroot
 
-if (WIN32)
+if("${CMAKE_HOST_SYSTEM}" MATCHES ".*Windows.*")
```

Comment:
https://cmake.org/cmake/help/v3.13/variable/WIN32.html?highlight=win32

Maar goed, jij checkt **host** en niet target system. Daar is wat voor te zeggen, in dit geval. Het is meer idiomatisch om `if(CMAKE_HOST_SYSTEM MATCHES "Windows")` te schrijven -- variable expansion is implicit, en de match is niet anchored en hoeft niet de hele string te matchen. Dus "ergens komt Windows voor in de host system" hoeft geen sterretjes.

Reply:
Er .. language settings :) If you specifically want to check the host system (and I guess that makes sense, since that's where the build is happening and you need to run python during the build, on the host) then that can be done more idiomatically as ..